openmetadata-ui/src/main/resources/ui/src/components/Users/Users.style.less

```diff
@@ -30,7 +30,14 @@
 
 .user-page-layout {
   height: @users-page-tabs-height;
-  .page-layout-rightpanel {
+
+  .user-layout-scroll {
+    height: @users-page-tabs-height;
+    overflow-y: auto;
+    overflow-x: hidden;
+  }
```

Comment:
This seems useless, we are giving height to back to back two elements which will cause two scrollbar in ui